tests/test_chargebee_automatic_fields.py

```diff
@@ -0,0 +1,66 @@
+from tap_tester import connections,runner
+from base import ChargebeeBaseTest
+
+class ChargebeeAutomaticFieldsTest(ChargebeeBaseTest):
+
+    def name(self):
+        return "chargebee_automatic_fields_test"
+
+    def automatic_fields_test_run(self):
+        """
+        Testing that all the automatic fields are replicated despite de-selecting them
+        - Verify that only the automatic fields are sent to the target.
+        - Verify that all replicated records have unique primary key values.
+        """
+
+        untestable_streams = {'quotes'} # For V2, we have 0 records for 'quotes' stream
```

Comment:
if it is common across all tests then can we define it in the base.py?

Reply:
Previously, Kyle has suggested us not to add skipping stream in the base file as in future someone would not know the skipped stream in that test case. [Reference](https://github.com/singer-io/tap-snapchat-ads/pull/15#discussion_r864843867)